src/core/lib/iomgr/combiner.c

```diff
@@ -50,17 +50,20 @@ int grpc_combiner_trace = 0;
     }                           \
   } while (0)
 
+#define STATE_UNORPHANED 1
```

Comment:
If there's UNORPHANED to represent 1, how about ORPHANED to represent 0? Makes the case statements below more readable.